src/bridge/test-fs.c

```diff
@@ -90,9 +90,9 @@ setup (TestCase *tc,
 
 static void
 setup_fsread_channel (TestCase *tc,
-                      const gchar *path)
+                      const gchar *path, gboolean binary)
```

Comment:
Please either use the boolean also on the API of `cockpit_fsread_open()` or use a string here.  The mix is weird.

Reply:
Boolean it is